tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "emitDeclarationOnly": true,
     "strict": true,
     "pretty": true,
+    "resolveJsonModule": true,
```

Comment:
We need this because new version of `@mdn/browser-compat-data` imports `.json` somewhere :)